### DISCUSSION TITLE
Big thanks...

### DISCUSSION THREAD

**Discussion by Scott-GT** [varszegimarcell/Optiplex-3020-Hackintosh-OpenCore#73] [General]
I've been using Hackintoshes for many years now but this has to have been the easiest build yet. Great guide.

My little Dell 3020M is rock solid. I've upgraded the CPU from an i3 to i7 4785T, 8GB of memory and a 250GB SSD. Next is adding a little M.2 A/E to PCIE to an old MacBook wifi card.

Those looking to upgrade the OS...it's running Monterey 12.6 without a problem.

- Follow the OpenCore guide regarding upgrading OpenCore (now 0.8.5)
- Use OCConfigCompare against the 8.5 sample and the original one from here. Copy/paste the handful of changes (I used Propertree)
- Follow the instructions here on running GenSMBIOS:
- Use iMac16,2 and this will update your config.plist
- Replace with the latest kexts

Job done.

Thanks for all for the hard work...lovely little machine.

**Comment by Xtendera** [comment]
Nice! Glad we could help.
